en/appendices/4-0-migration-guide.rst

```diff
@@ -457,3 +473,7 @@ View
   If you need to retain the former markup, a shimmed FormHelper can be found in
   `Shim plugin <https://github.com/dereuromark/cakephp-shim>`__ with the old
   behavior/generation (4.x branch).
+* ``FormHelper`` now sets the default step size to seconds for ``datetime``
```

Comment:
There is also a list of abstract [types here](https://github.com/cakephp/docs/blob/3.0/en/orm/database-basics.rst#data-types), and a mapping of types to input types [here](https://github.com/cakephp/docs/blob/3.0/en/views/helpers/form.rst#creating-form-controls) that need to be updated.